karma.conf.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+var depMain = function(name) {
+  var pkg = require(name + '/package.json');
+  var main = 'main' in pkg ? pkg.main : 'index.js';
+  return './node_modules/' + name + '/' + main;
+};
+
+// https://saucelabs.com/platforms
+var customLaunchers = {
+  sl_chrome: {
+    base: 'SauceLabs',
+    browserName: 'chrome',
+    version: '51'
+  },
+  sl_firefox: {
+    base: 'SauceLabs',
+    browserName: 'firefox',
+    version: '47'
+  },
+  sl_Safari_8: {
+    base: 'SauceLabs',
+    browserName: 'safari',
+    platform: 'OS X 10.10',
+    version: '8'
+  },
+  SL_Safari_10: {
+    base: 'SauceLabs',
+    browserName: 'safari',
+    platform: 'OS X 10.11',
+    version: '10'
+  },
+  SL_iOS_Safari_8: {
+    base: 'SauceLabs',
+    browserName: 'iphone',
+    version: '8.1'
+  },
+  SL_IE_9: {
```

Comment:
It's not case sensitive, i just missed it when copying snippets around. Ill make them all lower